dcpy/lifecycle/ingest/run.py

```diff
@@ -1,15 +1,17 @@
 import json
 from pathlib import Path
-import typer
-
+import shutil
 from dcpy.models.lifecycle.ingest import Config
 from dcpy.connectors.edm import recipes
 from . import configure, extract, transform, validate
 
-TMP_DIR = Path("tmp")
+INGEST_DIR = Path(".lifecycle") / "ingest"
```

Comment:
Had started trying to define some shared path'ing stuff up in [lifecycle](https://github.com/NYCPlanning/data-engineering/blob/2f0a75aa2f9bb60ae6d2f93d64be45000d63f354/dcpy/lifecycle/__init__.py#L6) FYI